Rakefile

```diff
@@ -9,6 +9,32 @@ task :test do
   sh "python -m tests.benchmark"
 end
 
+desc 'CI dependent task; tasks in parallel'
+task:scalable_test do
+  sh "docker-compose up -d | cat"
+  n_total_envs = `tox -l | wc -l`
+  circle_node_tot = ENV['CIRCLE_NODE_TOTAL'].to_i
+  n_envs_chunk = n_total_envs.to_i / circle_node_tot
+  env_limiter_one = 1
+  env_limiter_two = n_envs_chunk
+  puts circle_node_tot
+  begin
+    for node_index in 0..circle_node_tot
+      if ENV['CIRCLE_NODE_INDEX'].to_i == node_index then
+        if node_index >= 1 then
```

Comment:
why do we need this condition - what's special about the case `CIRCLE_NODE_INDEX == 0`?